Mapify/Locale.cs

```diff
@@ -52,7 +53,17 @@ public static string Get(string key)
             }
 
             Dictionary<string, string> localeDict = csv[locale];
-            return localeDict.TryGetValue(key.TrimStart(PREFIX_CHARS), out string value) ? value : MISSING_TRANSLATION;
+
+            if (localeDict.TryGetValue(key.TrimStart(PREFIX_CHARS), out string value)) {
+                return value;
+            }
+
+            // If there is no translation for this station's name, don't translate it.
+            if (key.StartsWith(STATION_PREFIX)) {
+                return key.Replace(STATION_PREFIX, "");;
+            }
```

Comment:
Should we have a more generic system for handling cases like this? I haven't looked into Katy's language-helper, but it might help here.

Reply:
I am in favour of using LangHelper, it's easy to setup and editor ready.

Reply:
I haven't used Language helper, so IDK. If we do want to use language helper, i suggest we make it a new issue.

Reply:
@Insprill Can you rephrase that? I'm not sure what I should change in this PR now.

Reply:
On second thought, what I suggested wouldn't be great either. I'll accept this for now, and we can develop a better solution soon, along with moving to Language Helper.